packages/nextjs/components/scaffold-eth/Faucet.tsx

```diff
@@ -21,22 +20,28 @@ export const Faucet = () => {
   const [sendValue, setSendValue] = useState("");
 
   const { chain: ConnectedChain } = useNetwork();
-  const provider = getLocalProvider(localhost);
+  const provider = useMemo(() => getLocalProvider(localhost), []);
```

Comment:
What if we move this outside/above of the component?

```
const provider = getLocalProvider(localhost);

export const Faucet = () => { 
//... 
}
```

Would it have the same effect?